app/components/post-meta.js

```diff
@@ -1,5 +1,18 @@
 import Ember from 'ember';
 
+/**
+  The post-meta component is a badge that presents a post's status
```

Comment:
Something is off here. I think the component is named incorrectly. It looks like we probably want to change `post-meta` to `post-status` and move the `post.status` binding into the component itself. This PR is small enough we can tackle that here if we'd like.